### PR TITLE
Menu changed

### PR DESCRIPTION
**Pull request by olahfemi**
I did more work on the menu change. Please review and merge.

### CODE REVIEW

**Comment by olahfemi** [comment]
Alright.
I'll check it out.
Thanks.

**Comment by olahfemi** [comment]
I've configured the `DATABASE_URL` environmental variable and used it to configure the database.

Also, the admin URL is still in the `url.py` file, on `line 18`. I've attached a screenshot of the contents of the file:
admin url

Please help deploy the site as early as possible. Mr. Donald needs it to work on **Monday** cos some announcement will be made and people will be visiting the `/unilag` page.

File: source_code/config/settings/local.py [hunk 1]
```diff
@@ -74,13 +75,19 @@
...
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'qweryacademy',
-        'USER': 'postgres',
-        'PASSWORD': '123456',
-        'HOST': 'localhost',
-        'PORT': '',
-    }
-}
+db_from_env = dj_database_url.config()
```
Comment (gbozee) [inline source_code/config/settings/local.py]: This isn't what was configured before. If you look at the top of this file, it is inheriting from `common`. in the `common.py` the `DATABASES` configuration is already configured and is expecting an environmental varibale `DATABASE_URL`. So for development, you just needed to set your envioronmental variable alone. You are not supposed to add any configuration to the `local.py` at all.

**Comment by olahfemi** [comment]
Alright.
I've deleted my configurations in `local.py'.
It's fixed now.
Thanks.

**Pull request merged**